src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx

```diff
@@ -128,88 +241,87 @@ export const WalletsAndKeys = () => {
             !walletBackedUp && !firstNotBackedUpRef.current;
 
           return (
-            <Menu
-              ref={firstNotBackedUp ? firstNotBackedUpRef : undefined}
-              key={idx}
-            >
-              <MenuItem
-                testId={`wallet-group-${idx + 1}`}
-                first
-                titleComponent={
-                  <MenuItem.Title
-                    text={`${i18n.t(
-                      wallet.type === KeychainType.HdKeychain
-                        ? 'settings.privacy_and_security.wallets_and_keys.recovery_phrase_label'
-                        : wallet.type === KeychainType.HardwareWalletKeychain
-                        ? 'settings.privacy_and_security.wallets_and_keys.hardware_wallet_label'
-                        : 'settings.privacy_and_security.wallets_and_keys.private_key_label',
-                    )} ${
-                      wallet.type === KeychainType.HdKeychain
-                        ? walletCountPerType.hd
-                        : wallet.type === KeychainType.HardwareWalletKeychain
-                        ? walletCountPerType.hw
-                        : walletCountPerType.pk
-                    }`}
-                  />
-                }
-                labelComponent={
-                  <Inline alignVertical="center" space="4px">
-                    {importedLabel ? (
-                      <Text
-                        color={walletBackedUp ? 'labelTertiary' : 'red'}
-                        size="12pt"
-                        weight={walletBackedUp ? 'medium' : 'bold'}
-                      >
-                        {importedLabel}
+            <WalletsAndKeysContextMenu key={idx} wallet={wallet}>
+              <Menu ref={firstNotBackedUp ? firstNotBackedUpRef : undefined}>
+                <MenuItem
+                  testId={`wallet-group-${idx + 1}`}
+                  first
+                  titleComponent={
+                    <MenuItem.Title
+                      text={`${i18n.t(
+                        wallet.type === KeychainType.HdKeychain
+                          ? 'settings.privacy_and_security.wallets_and_keys.recovery_phrase_label'
```

Comment:
why not `{t('blabla')}` here?

Reply:
idk I think I created the `t` after doing this and missed later, just updated both